src/requirements/data/colleges/ag.ts

```diff
@@ -8,7 +8,8 @@ const calsCreditsRequirement: CollegeOrMajorRequirement = {
     + 'Biological Sciences, Biology & Society, Earth and Atmospheric Sciences, Information Science, Nutritional Science, '
     + 'and The Department of Statistics and Data Science.',
   source: 'https://cals.cornell.edu/undergraduate-students/student-services/degree-requirements/graduation-requirements',
-  checker: (course: Course): boolean => ['AG', 'BU'].includes(course.acadGroup),
+  checker: (course: Course): boolean => ['AG'].includes(course.acadGroup)
+  || ['AEM', 'BIOEE', 'BIOMG', 'BIOMI', 'BIONB', 'BSOC', 'EAS', 'INFO', 'NS', 'STSCI'].includes(course.subject),
```

Comment:
Should we be doing this the same way as we do A&S now?

Reply:
i don't think so. for arts i think we just check any distribution that ends in a '-as'. but for this req, it says it has to be in CALS or the listed certain departments.
so like 'into to china' would fulfill the a&s req (because it's a -as class), but it shouldn't for this one.

Reply:
hm i tried doing it the same way using '-AG' but certain departments like info don't get counted, even tho info sci is offered in CALS. the catalogDistr for info sci is '-AS'

Reply:
Oops you're totally doing it right, my bad.